### PR TITLE
Testing gatsby-source-custom-api

### PR DESCRIPTION
**Pull request by dmotan**

### CODE REVIEW

File: gatsby_fp/gatsby-config.js [hunk 1]
```diff
@@ -7,11 +7,88 @@ module.exports = {
...
+            country: String
+            major: String
+        `,
+          member: `
+            org: String
+            status: String
+            orgabbr: String
+        `,
+          intellcont: `
+            contype: String,
```
Comment (mf-stevens) [inline gatsby_fp/gatsby-config.js:69]: Is this comma supposed to be here?

File: gatsby_fp/gatsby-config.js [hunk 2]
```diff
@@ -7,11 +7,88 @@ module.exports = {
...
+            issue: String
+            dty_pub: String
+            dty_acc: String
+            dty_sub: String
+            web_address: String
+            intellcont_auth: [intellcont_auth]
+        `,
+          intellcont_auth: `
+            mname: String
+                fname: String
```
Comment (mf-stevens) [inline gatsby_fp/gatsby-config.js:87]: Inconsistent indentation

File: gatsby_fp/gatsby-config.js [hunk 3]
```diff
@@ -7,11 +7,88 @@ module.exports = {
...
+      resolve: "gatsby-source-custom-api",
       options: {
-        apis: [
-          `https://${process.env.PEOPLEAPI_USER}:${process.env.PEOPLEAPI_PASS}@${process.env.PEOPLEAPI_HOST}`,
-        ],
+        url: `https://${process.env.PEOPLEAPI_USER}:${process.env.PEOPLEAPI_PASS}@${process.env.PEOPLEAPI_HOST}`,
+        rootKey: 'profiles',
+        schemas: {
+          profiles: `
+          pf_username: String
```
Comment (mf-stevens) [inline gatsby_fp/gatsby-config.js:16]: Inconsistent indentation

File: gatsby_fp/gatsby-config.js [hunk 4]
```diff
@@ -7,11 +7,88 @@ module.exports = {
...
+          photo_base64: String
+          notable_courses: String
+          school: String
+          ses_department: String
+          service_university: [service_university]
+          service_professional: [service_professional]
+          education: [education]
+          member: [member]
+          intellcont: [intellcont]
+        `,
```
Comment (mf-stevens) [inline gatsby_fp/gatsby-config.js:45]: Inconsistent indentation (this backtick should have the same amount of indentation as line 15)

File: gatsby_fp/gatsby-config.js (hunk 1 above)
Comment (dmotan) [reply gatsby_fp/gatsby-config.js:69]: I removed it

File: gatsby_fp/gatsby-config.js (hunk 3 above)
Comment (dmotan) [reply gatsby_fp/gatsby-config.js:16]: I fixed it, it's not the problem

File: gatsby_fp/gatsby-config.js (hunk 4 above)
Comment (dmotan) [reply gatsby_fp/gatsby-config.js:45]: I fixed it, it's not the problem

**Comment by dmotan** [comment]
This was not intended to be merged, it was created to test Netlify behaviour

**Pull request closed without merging**